inst/shiny/modules/tab_data.R

```diff
@@ -1,52 +1,62 @@
-#' The data navbar tab loads a dummy data set by default.
 #' The user can upload a new data set by clicking the "Browse" button.
 #' The data set must be in CSV format and contain the following columns:
 #' STUDYID, USUBJID, ANALYTE, PCSPEC, DOSEFRQ, DOSNO, AFRLT, ARRLT, NRRLT, NFRLT,
 #' AVAL, AVALU, ROUTE, DOSEA, AGE
 
 source(system.file("/shiny/modules/input_filter.R", package = "aNCA"))
+source(system.file("/shiny/modules/column_mapping.R", package = "aNCA"))
 
 tab_data_ui <- function(id) {
   ns <- NS(id)
 
-  sidebarLayout(
-    sidebarPanel(
-      actionButton(ns("login"), "Login and Select File Path"),
-      br(), br(), br(),
-      # Local upload option
-      fileInput(
-        ns("local_upload"),
-        width = "60%",
-        label = NULL,
-        placeholder = "CSV rds",
-        buttonLabel = list(icon("folder"), "Upload File..."),
-        accept = c(".csv", ".rds")
+  navset_pill(
+    id = ns("data_navset"),
+    nav_panel("Raw Data Upload",
+      card(
+        "Upload your PK dataset in .csv format",
+        # Local upload option
+        fileInput(
+                  ns("local_upload"),
```

Comment:
nitpick: indentation

Reply:
the indentation was done to pass the lintr- this is what it made me change it to so I'm hesitant to change it